assets/src/edit-story/elements/media/frame.js

```diff
@@ -42,7 +42,7 @@ function MediaFrame({ element: { id } }) {
     setEditingElement,
   ]);
   const getHandleClick = useDoubleClick(handleSingleClick, handleDoubleClick);
-  return <Element onClick={getHandleClick(id)} />;
+  return <Element data-testid={dataTestid} onClick={getHandleClick(id)} />;
```

Comment:
Why not set the test ID directly here instead of passing it as a prop? Seems like an uncommong thing to do.

Reply:
I was using it here `<MediaFrame dataTestid="videoFrame" element={element} />` so for image what would be different `imageFrame`, but eventually decided that it's not needed (will just use higher `frameElement`).